### PR TITLE
bump version to 2.0.0-RC1

### PR DESCRIPTION
**Pull request by nobodyiam**
## What's the purpose of this PR

bump version to 2.0.0-RC1

## Brief changelog

1. bump version to 2.0.0-RC1
2. add 1.9.2 changes note(which should be done when 1.9.2 is released)

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [x] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [x] Update the [`CHANGES` log](https://github.com/ctripcorp/apollo/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by nisiyong** [comment]
I think PMC members or committers could create the branch in the official repo to do this stuff, it will be easy for others to check out and test it.

**Comment by nobodyiam** [comment]
@nisiyong 

After this pr is merged, the 2.0.0-RC1 version would be pre-released in the [release page](https://github.com/apolloconfig/apollo/releases) and a vote would be created in the [discussions](https://github.com/apolloconfig/apollo/discussions). PMC members or committers could then check the artifacts and release notes to decide +1 or -1 to this release.

**Review by lepdou — APPROVED** [review]
LGTM

**Pull request merged**
